app/v2/govuk_alerts/get_broadcasts.py

```diff
@@ -1,8 +1,11 @@
 from flask import jsonify
 
+from app.dao.broadcast_message_dao import dao_get_all_broadcast_messages
 from app.v2.govuk_alerts import v2_govuk_alerts_blueprint
 
 
 @v2_govuk_alerts_blueprint.route('')
 def get_broadcasts():
-    return jsonify({})
+    broadcasts = dao_get_all_broadcast_messages()
+    broadcasts_dict = {"alerts": [broadcast._asdict() for broadcast in broadcasts]}
```

Comment:
rather than calling the sqlalchemy builtin _asdict, we should manually define the json that we will return. That way we can add, remove, and rename fields more easily. Also this will solve the datetime problem. 

In existing serialize functions in models.py we use `some_time.strftime(DATETIME_FORMAT)` and `get_dt_string_or_none(some_nullable_time)`, with `from app.utils import DATETIME_FORMAT, get_dt_string_or_none`. We should follow the same patterns here.

Reply:
@leohemsted yeah, I looked at get_dt_string_or_none before, but that didn't work mid-query. I thought about doing `cast` in query to change data types, but couldn't find good examples for Datetime to String.

 I also thought about creating a `serialize_for_gov_uk_alerts()` method in BroadcastMessage in `models.py`, but we needed some data outside of this model, namely `channel` from `ServiceBroadcastSettings`.
 
 I now did the json definition in `get_broadcasts` like you suggested and transformed the datetimes there, check if it meets the bill :). My only reservation is that doing it this way is probably slower than doing it from inside the query, but probably it won't matter that much, as hopefully we won't have enough alerts for it to matter?